bot.py

```diff
@@ -1160,9 +1235,26 @@ async def cleanupInactiveGames(self):
     async def beforecleanupInactiveGames(self):
         await self.bot.wait_until_ready()
 
+    # Start the vote timer
+    @commands.command(name='voteTimer', aliases=['vt', 'votetimer'], help=f'Start a countdown to voting time.\n\nUsage: {COMMAND_PREFIX}votetimer <time string>\n\nTime string can look like: "5 minutes 30 seconds" or "5:30" or "5m30s"')
+    async def start_timer(self, ctx):
+        await self.perform_action_reporting_errors(self.votetimer.start_timer, ctx)
+
+    # Stop an ongoing vote timer
+    @commands.command(name='stopVoteTimer', aliases=['svt', 'stopvotetimer'], help=f'Stop an existing countdown to voting.\n\nUsage: {COMMAND_PREFIX}stopvotetimer')
+    async def stop_timer(self, ctx):
+        await self.perform_action_reporting_errors(self.votetimer.stop_timer, ctx)
+
+    async def perform_action_reporting_errors(self, action, ctx):
+        try:
+            message = await action(ctx)
+            if message != None:
+                await ctx.send(message)
+
+        except Exception as ex:
+            await ctx.bot.sendErrorToAuthor(ctx)
 
-COMMAND_PREFIX = os.getenv('COMMAND_PREFIX') or '!'
 bot = botcBot(command_prefix=COMMAND_PREFIX, intents=intents, description='Bot to manage playing Blood on the Clocktower via Discord')
-bot.add_cog(Setup(bot))
-bot.add_cog(Gameplay(bot))
-bot.run(TOKEN)
+bot.add_cog(SetupCog(bot))
```

Comment:
Renaming these cogs to have Cog in them has made the !help menu ugly, because it now says “Cog” too. There’s probably a way to have the help category name not be the literal class name